browser/net/brave_ad_block_tp_network_delegate_helper.cc

```diff
@@ -223,6 +221,12 @@ int OnBeforeURLRequest_AdBlockTPPreWork(const ResponseCallback& next_callback,
     return net::OK;
   }
 
+  DCHECK(ctx->browser_context);
+  // DoH or standard DNS quries won't be routed through Tor, so we need to skip
+  // it.
+  if (ctx->browser_context->IsTor()) {
+    return net::OK;
```

Comment:
This is going to disable adblocker in tor windows. What we need to do here is an adblocking pass without CNAME resolving, e.g. we can directly call `ShouldBlockAdWithOptionalCname` with empty CNAME

Reply:
fixed